skillner/skillner_flow.py

```diff
@@ -0,0 +1,230 @@
+"""
+Flow to train an NER model to extract skills from jobs.
+
+python skillner_flow.py --package-suffixes=.txt run
+"""
+import os
+from pathlib import Path
+
+os.system(f"pip install -r {Path.cwd()}/skillner_requirements.txt 1> /dev/null")
+import boto3
+from dotenv import load_dotenv
+from metaflow import FlowSpec, Parameter, step
+from wasabi import msg
```

Comment:
👀

Reply:
honestly really good to know that its a good alternative to logger